### ISSUE TITLE
workbox - 404 for html and json files

### ISSUE THREAD

**Issue by silverbackdan** [nuxt-community/modules#103]
Is there an update in the nuxt rc3 which means some file types cannot be served from the .nuxt/dist directory?

Workbox is generating index.sa.html index.ssr.html and vue-ssr-client-manifest.json and I can see them in the directory.

When running I'm getting 404s though

![screen shot 2017-08-01 at 17 51 34](https://user-images.githubusercontent.com/25174262/28836732-141e6a4e-76e2-11e7-8d50-dcd2d937d70a.png)

**Comment by pi0** [comment]
Hi @silverbackdan . Thanks for report. Actually i were are of this but so busy :( Will fix it soon ;)

**Comment by silverbackdan** [comment]
I can imagine! Some really superb changes happening with Nuxt - if I could figure it out I'd create a PR to give you one less job but I'm not sure I can. Thanks as always Pooya :)

**Comment by silverbackdan** [comment]
Just a bit of info (you already know this though). Found the line preventing some files from being public.
https://github.com/nuxt/nuxt.js/blob/69e645037746ef416d49594521820d12318f2e0d/lib/core/renderer.js#L182

Do this pages that are resulting in 404s need to be public and requires an update in Nuxt itself or does this plugin need to ignore them to ignore list?
```
globIgnores: ['**/server-bundle.json','**/vue-ssr-client-manifest.json','**/index.{spa,ssr}.html']
```

I could work on a PR for the most appropriate if you'd like.

If the Nuxt files should be public I could work on a way to prepend 'public.' to the files perhaps and then adjust the check.

If they need to be ignored I can raise a very simple PR

Otherwise I'll just leave this with you for when the more important bits have been resolved. :)

**Comment by pi0** [comment]
Thanks @silverbackdan for your checks and sorry again for delay :( :)) I've limited pre-cached assets to only css and js files. So should be fixed with workbox@1.1.3
